Vagrantfile

```diff
@@ -22,7 +22,8 @@ end
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   # For LXC. VirtualBox hosts use a different box, described below.
-  config.vm.box = "fgrehm/trusty64-lxc"
+  config.vm.box = "developerinlondon/ubuntu_lxc_xenial_x64"
```

Comment:
This tested.

Reply:
`vagrant ssh` works just fine, but provision failed with the following error:
```
Job for redis-server.service failed because the control process exited with error code. See "systemctl status redis-server.service" and "journalctl -xe" for details.
invoke-rc.d: initscript redis-server, action "start" failed.
dpkg: error processing package redis-server (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 redis-server
E: Sub-process /usr/bin/dpkg returned an error code (1)
Traceback (most recent call last):
  File "./lib/provision.py", line 96, in <module>
    subprocess.check_call(["sudo", "apt-get", "install", "-y", "lsb-release"])
  File "/usr/lib/python3.5/subprocess.py", line 581, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['sudo', 'apt-get', 'install', '-y', 'lsb-release']' returned non-zero exit status 100
```

I have tried another vm box, `aravind/xenial-lxc-amd64` with the same error.
Upon digging some forums I found that people have been having problem with installing redis-server on Xenial: https://unix.stackexchange.com/questions/351668/cannot-install-redis-server, with a bug report: https://bugs.launchpad.net/ubuntu/+source/redis/+bug/1663911.

Reply:
Fixed in the second commit of this PR.

Reply:
I'm not sure how long has this redis-on-xenial bug been hiding.